UnitySDK/Assets/ML-Agents/Editor/Tests/Sensor/FloatVisualSensorTests.cs

```diff
@@ -0,0 +1,105 @@
+using NUnit.Framework;
+using UnityEngine;
+using MLAgents.Sensor;
+
+namespace MLAgents.Tests
+{
+    public class Float2DSensor : ISensor
```

Comment:
This is probably usable on its own, but I can't think of a good usecase in our current examples. It would be cool to use for occupancy maps (see https://www.gdcvault.com/play/1267/(307)-Beyond-Behavior-An-Introduction starting around 13:00)

Reply:
Maybe we can most it out of tests and to a fully-supported component when we make a hide-and-seek demo.